cmd/exec/operations.go

```diff
@@ -527,11 +527,12 @@ func executeEvent(url string) (string, error) {
 		}
 
 		if resp.StatusCode != http.StatusOK {
-			return "", errors.New("Bad request")
+			log.Fatalf("Failed to send event: %v", err)
```

Comment:
Same as above: the function should return the error and escalate it to the caller. It should be the caller's decision whether to fail or try something else.